src/hooks/useDeleteMeeting.ts

```diff
@@ -1,19 +1,15 @@
-import { useMutation, useQueryClient } from '@tanstack/react-query';
+import { useMutation } from '@tanstack/react-query';
 import axios from 'axios';
 import { deleteMeeting } from '../api/meeting';
-import { useLocation, useNavigate } from 'react-router-dom';
 
-const useDeleteMeeting = () => {
-  const queryClient = useQueryClient();
-  const navigate = useNavigate();
-  const location = useLocation();
+interface UseDeleteMeetingProps {
+  onSuccess: () => void;
+}
+const useDeleteMeeting = ({ onSuccess }: UseDeleteMeetingProps) => {
   return useMutation({
     mutationFn: (id: string) => deleteMeeting(id),
     onSuccess: () => {
```

Comment:
없어도 정상작동 하나요?

Reply:
일단 onSuccess값을 무조건 다 넘기고 있기 때문에 옵셔널처리는 하지 않았고
없어도 api는 정상적으로 호출이 되긴합니다.

Reply:
```suggestion
    onSuccess,
```

예외처리라는게 onSuccess?: () => void; 말씀하시는거면 안쓰는경우 생기면 바꾸면 될 것 같아요

Reply:
넵

Reply:
onSuccess: () => {
          onSuccess();
    },
한줄밖에 선택이 안되어서 그랬는데 위에 3줄을 onSuccess,로 바꾸는걸 제안드린거였어요